### PR TITLE
Added CAF note for DNS

### PR DESCRIPTION
**Pull request by gilherau**
Added reference to the cloud adoption framework (CAF) article on how to do private endpoints at scale with a specific note about Azure Policies to automate DNS entries.

### CODE REVIEW

**Comment by gilherau** [comment]
I can't seem to be able to get the right relative link working for https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/private-link-and-dns-integration-at-scale

I have it as "azure/cloud-adoption-framework/ready/azure-best-practices/private-link-and-dns-integration-at-scale" but it's not letting it through? Can you help me resolve this?

**Comment by Court72** [comment]
Hi @gilherau

Try: `[Private Endpoint and DNS configuration at Scale](/azure/cloud-adoption-framework/ready/azure-best-practices/private-link-and-dns-integration-at-scale)`

**Comment by Court72** [comment]
@shsagir 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by gilherau** [comment]
#sign-off

File: data-explorer/security-network-private-endpoint.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ Clients on a VNet using a private endpoint should use the same connection string
 
 By default, Azure Data Explorer creates a [private DNS zone](/azure/dns/private-dns-overview) attached to the VNet with the necessary updates for the private endpoints. However, if you're using your own DNS server, you may need to make more changes to your DNS configuration.
 
+> [!IMPORTANT]
+> We recommend that your Azure tenant be configured to align with the recommendations found in this [Private Endpoint and DNS configuration at Scale](/azure/cloud-adoption-framework/ready/azure-best-practices/private-link-and-dns-integration-at-scale) article from the Cloud Adoption Framework. Azure Data Explorer is one of many services in Azure that leverages Private Endpoints and you will benefit from following this guidance. Specifically, this article will instruct you on how to automate Private DNS entry creation using Azure Policies. This will prove invaluable as you scale out your Azure Data Explorer footprint.
```
Comment (shsagir) [inline data-explorer/security-network-private-endpoint.md:63]:
```suggestion
> For optimal configuration, we recommend that you align your to align with the recommendations in the [Private Endpoint and DNS configuration at Scale](/azure/cloud-adoption-framework/ready/azure-best-practices/private-link-and-dns-integration-at-scale) Cloud Adoption Framework article. Use the information in this article provides to automate Private DNS entry creation using Azure Policies, removing the need to manually add and delete DNS records as you expand your Azure Data Explorer deployment.
```

@gilherau Please check to make sure I've retained the intent of the note. Thx

**Review by gilherau — COMMENTED** [review]
All good

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**